full_snapshot.proto

```diff
@@ -176,6 +187,10 @@ message Replication {
   int64 apply_byte_lag = 23;
   google.protobuf.Timestamp replay_timestamp = 24;
   int64 replay_timestamp_age = 25; // in seconds
+
+  // Piggyback txid data
+  int64 current_xact_id = 30;
+  int64 next_multi_xact_id = 31;
```

Comment:
[note for reviewer]
This is "Replication" message and it's not exactly fitting to put these data in. However, in our pganalyze app side, we are storing this data among replication stats data, therefore it makes sense to piggyback and add them here.

Reply:
I guess this answers my question on the other PR. I don't think this is a great idea, but okay.

Reply:
Actually, putting this out of `Replication` message won't change much of how I can put things together, so let me tweak.